### PR TITLE
fix(charts/cloudflare-tunnel-remote): remove invalid secret ref key

### PR DESCRIPTION
**Pull request by uhthomas**
It is not valid to specify individual keys with 'envFrom'. Instead, create environment variables for all secret key value pairs.

Fixes: #46

### CODE REVIEW

**Comment by denzhel** [comment]
Can you please merge this ?

**Comment by uhthomas** [comment]
Looks like @obezuk is out of office for another couple of weeks. Will ask around and see if anyone else can merge this.

**Review by jcsf — CHANGES_REQUESTED** [review]

File: charts/cloudflare-tunnel-remote/templates/deployment.yaml [hunk 1]
```diff
@@ -29,15 +29,14 @@ spec:
...
           - tunnel
-          # In a k8s environment, the metrics server needs to listen outside the pod it runs on. 
+          # In a k8s environment, the metrics server needs to listen outside the pod it runs on.
           # The address 0.0.0.0:2000 allows any pod in the namespace.
           - --metrics
           - 0.0.0.0:2000
           - run
         envFrom:
         - secretRef:
             name: {{ include "cloudflare-tunnel-remote.fullname" . }}
```
[the remark this answers is not in the document]
Comment (uhthomas) [reply charts/cloudflare-tunnel-remote/templates/deployment.yaml:39]: I had the same thought and that was initially what I was going to do, but wrote it this way purposefully as it seemed to be closer to what the author intended. Happy to make this change though. Give me a second.

**Review by jcsf — APPROVED** [review]

**Review by chungthuang — APPROVED** [review]

**Comment by denzhel** [comment]
Please merge :) ?

**Comment by evfurman** [comment]
Can we please get this merged? 🙏

**Comment by denzhel** [comment]
Can we get this merged please ?

**Review by tvarney — APPROVED** [review]

**Review by larcher — APPROVED** [review]

**Pull request merged**